modules/dnn/include/opencv2/dnn/dnn.hpp

```diff
@@ -100,6 +100,9 @@ CV__DNN_INLINE_NS_BEGIN
     CV_EXPORTS std::vector< std::pair<Backend, Target> > getAvailableBackends();
     CV_EXPORTS_W std::vector<Target> getAvailableTargets(dnn::Backend be);
 
+    // enable model diagnostics
+    CV_EXPORTS void setDiagnosticsRun(bool enabled);
```

Comment:
Please mention reading / loading stage of DNN models in the name and documentation string.
Current generic name may confuse users.